hera_qm/firstcal_metrics.py

```diff
@@ -0,0 +1,579 @@
+"""
+FirstCal metrics
+"""
+import matplotlib
+import matplotlib.pyplot as plt
+plt.switch_backend('Agg')
```

Comment:
Is this line still necessary with the travis fix? It's messing up my environment after I import anything from hera_qm.